docs/what_makes_securedrop_unique.rst

```diff
@@ -0,0 +1,105 @@
+What Makes SecureDrop Unique
+============================
+
+SecureDrop attempts to solve or mitigate several problems journalists and sources
+have faced in recent legal investigations, attacks from state actors, and other
+threats to the confidentiality of communications.
+
+No third parties that can secretly be subpoenaed
+------------------------------------------------
+
+For decades, there were very few leak prosecutions in the United States in large
+part because the government would have to subpoena reporters to testify against
+a source to get a conviction. That proved incredibly difficult, if not impossible,
+when reporters regularly refused to testify and threatened to go to jail rather
+than betray a source.
+
+More recently, there have been a record number of leak prosecutions largely because
+the government has learned they don’t need reporters to testify against their
+sources anymore. Instead, they can just secretly subpoena third party services
+like Google or AT&T or Verizon or Facebook and get a treasure trove of digital
+information on reporters and sources’ communications. For example, the Associated
+Press had twenty of their phone lines subpoenaed without their knowledge in order
+to identify a source. The government also got a warrant for Fox News reporter James
+Rosen’s Gmail account without him knowing. In both cases, their alleged sources
+were prosecuted, even though journalists never directly divulged their sources.
+
+SecureDrop completely eliminates third parties from the equation and puts the
+power to challenge such cases back in the hands of reporters. The journalist and
+source communicate exclusively through one server that the news organization owns
+and sits on their property, so any legal order for information must go directly
+to the news organization rather than Google or AT&T. The news organization again
+has the power to contest the order or refuse to comply if they so wish.
+
+Limits the metadata trail as much as possible
+---------------------------------------------
+
+In many leak cases, the metadata of a journalist's communications—where you’re
+located, who you’re talking to, when you’re talking to them, and how often—can
+lead to trouble just as much as the actual content of your conversations.
+
+With SecureDrop, even if a government serves a court order directly to a news
+organization to compel the disclosure of information, SecureDrop logs much less
+information than email providers or phone companies do.
+
+The source can only log into SecureDrop through the Tor Browser, which masks the
+source’s IP address to begin with, so there is no indication who the source is
+(unless they disclose it) and where they are sending information from. The Tor IP
+address, the computer and the browser type that the source is using is not logged
```

Comment:
Oxford comma time?

"The Tor IP address, the computer **,** and the browser type ..."